src/Codeception/Test/Unit.php

```diff
@@ -130,7 +130,7 @@ public function getReportFields()
         ];
     }
 
-    public function getDependencies()
```

Comment:
getDependencies is a more interesting case. Conflict wasn't cause by a change of signature.
Codeception had this method first, PhpUnit introduced it in 7.2.0, so fixing signature could cause interesting bugs.
https://github.com/sebastianbergmann/phpunit/commit/66a0765da30140eb9618b50cbeddd6223eccdf4a#diff-511c90242d5589e941582006e0485184R964

Reply:
One solution could be to constrain codeception's phpunit version up to 7.1 until we drop the support of php `5.x`.